src/frameworks/frameworks.js

```diff
@@ -51,12 +51,15 @@ cwc.framework.External = {
 /**
  * Internal frameworks used for the preview.
  * @enum {!Object.<string>|string}
+ * @export
  */
 cwc.framework.Internal = {
   EV3: '/frameworks/internal/ev3_framework.js',
   MBOT: '/frameworks/internal/mbot_framework.js',
   MBOT_RANGER: '/frameworks/internal/mbot_ranger_framework.js',
-  MESSENGER: '/frameworks/internal/messenger_framework.js',
+  // MESSENGER is used in tests and needs to be exported.
+  // The single quotes are the equivalent of @export for keys in a hash literal
+  'MESSENGER': '/frameworks/internal/messenger_framework.js',
```

Comment:
A mix of single quotes and no quotes are problematic for the full compiler flags.
Please remove the single quotes and @export here and use the path as hard-coded value inside the test.
After changing this to .deps support this should be solved automatically.